SkimsAUX/workdir/prod/80X_crab_example/MultiCrab3.py

```diff
@@ -20,7 +20,7 @@
 workArea = 'crabProdv10p0'
 outDir = '/store/group/lpcsusyhad/Stop_production/Spring16_80X_Oct_2016_Ntp_v10X_AK8_QGL'
 Pubname = 'Spring16_80X_Oct_2016_Ntp_v10p0_AK8_QGL'
```

Comment:
the time stamp and version need to be changed.
Oct_2016 --> Nov_2016
v10p0 --> v11p0
AK8_QGL --> new_IDs